src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -126,6 +126,7 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
       setStripeStyle({
         width: activeTab?.getBoundingClientRect().width,
         left: (activeTab as HTMLElement)?.offsetLeft,
+        top: (activeTab as HTMLElement)?.getBoundingClientRect().height - 2,
```

Comment:
Whats that for?

Reply:
`top` needs to be set in JS now
`2` is the height of the stripe